similarity/manage.py

```diff
@@ -0,0 +1,147 @@
+from __future__ import print_function
+from flask.cli import FlaskGroup
+import click
+
+import webserver
+import metrics
+from operations import HybridMetric
+import db
+
+from sqlalchemy import text
+
+PROCESS_BATCH_SIZE = 10000
+
+cli = FlaskGroup(add_default_commands=False, create_app=webserver.create_app_flaskgroup)
+
+
+@cli.command()
+def init_similarity():
+    click.echo('Copying data')
+    with db.engine.begin() as connection:
+        query = text("""
+        INSERT INTO similarity(id)
+             SELECT id
+               FROM lowlevel
+        """)
+        connection.execute(query)
+
+
+def _get_recordings_without_similarity(connection, name, batch_size):
+    query = text("""
+        SELECT id
+          FROM similarity
+         WHERE %(metric)s IS NULL
+         LIMIT %(limit)s
+    """ % {'metric': name, 'limit': batch_size})
+    result = connection.execute(query)
+    rows = result.fetchall()
+    if not rows:
+        return []
+    ids = zip(*rows)[0]
+    return ids
+
+
+def _update_similarity(connection, name, row_id, vector, isnan=False):
+    value = '[' + ', '.join(["'NaN'::double precision"] * len(vector)) + ']' if isnan else str(list(vector))
+    query = text("""
+        UPDATE similarity
+           SET %(metric)s = %(value)s
+         WHERE id = %(id)s
+    """ % {'metric': name, 'value': 'ARRAY' + value, 'id': row_id})
+    connection.execute(query)
+
+
+@cli.command(name="add-metric")
+@click.argument("name")
```

Comment:
is there a way of adding all metrics at once without running this command many times?